demo/backend/advanced/community/elements/bottom-sheet/index.xml.njk

```diff
@@ -37,4 +37,174 @@ hv_title: "Bottom Sheet"
       trigger="press"
     />
   {%- endcall %}
+
+  <bottom-sheet:bottom-sheet
```

Comment:
Prior to merging, it would be worth breaking up this XML in separate files (see `backend/advanced/community/behaviors/share` for example)